src/components/toast/Toast.jsx

```diff
@@ -0,0 +1,14 @@
+import { Slide, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+export default function Toast({ message, type }) {
+  return toast(message, {
+    position: "bottom-right",
+    autoClose: 5000,
```

Comment:
you can give smaller timer as api is not taking that much time, like 1000 or 2000

Reply:
ya i had same thought, thanks, I changed it to 3 secs